Makefile

```
@@ -17,7 +17,7 @@ docker-run:
 	docker run -it --rm --name $(DOCKER_IMAGE_NAME)-01 $(DOCKER_IMAGE_NAME)
 
 test:
-	npx tsx --test ${TEST_FILES}
+	NODE_ENV=test npx tsx --test ${TEST_FILES}
```

I have another suggestion for this:

```suggestion
	DB_DRIVER=sqlite DB_FILENAME=":memory" npx tsx --test ${TEST_FILES}
```

I think this will work as well. Slightly prefer this right now because I want to do something more sophisticated a little later.